arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -720,6 +720,25 @@ class StressNGParams:
         ]
     ]
 
+    # The workdir and cleanup items are plugin-internal parameters that are not passed
+    # to the stress-ng command
+    workdir: typing.Annotated[
+        typing.Optional[str],
+        schema.name("Working Dir"),
+        schema.description(
+            "Directory in which stress-ng will be executed "
+            "(for example, to target a specific volume)"
+        ),
+    ] = None
```

Comment:
You like `None` better than, say, `"."`?

Reply:
This is old code, but it looks like the default is currently set explicitly in the plugin code, and it would be better to set it here, so I'll change that.

Reply:
In point of fact, if the current working directory is unspecified (i.e., `None`), then it defaults to the the parent process's current working directory, so `"."` is as good as `None` so long as nothing in the call chain tries to "fix" it...but it's probably good to be explicit, and I think it's nicer to omit the conditional/fixup code.